### PR TITLE
[Azure Stack] Add DRP Admin module

### PR DESCRIPTION
**Pull request by viananth**
## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by viananth** [comment]
PS Sign job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/146/

**Comment by bganapa** [comment]
11:07:16   Exception calling "WriteAllText" with "3" argument(s): "There is not enough space on the disk.

File: src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/Azs.Deployment.Admin.psd1 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            LicenseUri   = 'https://aka.ms/azps-license'
+
+            # A URL to the main website for this project.
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
+
+            # A URL to an icon representing this module.
+            # IconUri = ''
+
+            # ReleaseNotes of this module
+            ReleaseNotes = '
```
Comment (bganapa) [inline src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/Azs.Deployment.Admin.psd1]: Add more description here

Comment (bganapa) [reply src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/Azs.Deployment.Admin.psd1]: @shriramnat pls add the text here

File: src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/ChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 0.1.0
+* Initial version of powershell for Deployment resource provider admin operations
+* supports operations for deploying resource provider
```
Comment (bganapa) [inline src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/ChangeLog.md]: needs more description here as well

File: src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/Help/Get-AzsActionPlanAttempt.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+## DESCRIPTION
+Gets or lists the action plan attempts
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (bganapa) [inline src/StackAdmin/Azs.Deployment.Admin/Module/Azs.Deployment.Admin/Help/Get-AzsActionPlanAttempt.md]: example missing

**Comment by msJinLei** [comment]
Why do you want to add the new module to AzureRM since we no long update it?

**Comment by bganapa** [comment]
@msJinLei This is for AzureStack Admin. The module is getting added to the AZURESTACK uber module, We are still dependant on AzureRM branch. The release is scoped only to AzureStack module and not AzureRM. This should be the last release based on the AzureRM, as we are channeling our efforts towards Az autogenerator for the admin modules. We had a meeting with Eric and Dingmeng yesterday. Please sync up if needed.

**Comment by shriramnat** [comment]
@bganapa : here is teh description
The Deployment Resource Provider (DRP) enables orchestrated deployments of Resource Providers to Azure Stack. These commands interact with the Azure Resource Manager layer to interact with DRP.

**Comment by bganapa** [comment]
@markcowl  This should be good to merge.  The CI is failing with 
11:26:42 Unhandled Exception: System.IO.IOException: There is not enough space on the disk.

Another ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign-esrp/147/

**Pull request merged**